<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JSON Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_File"/>
   <addaction name="actionOpen_File"/>
   <addaction name="separator"/>
   <addaction name="actionDelete_File"/>
  </widget>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/open_doc.png</normaloff>:/icons/open_doc.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="toolTip">
    <string>Choose a file to open...</string>
   </property>
  </action>
  <action name="actionDelete_File">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete_File</string>
   </property>
   <property name="toolTip">
    <string>Delete the current file...</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/save_doc.png</normaloff>:/icons/save_doc.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="toolTip">
    <string>Click to save your file...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
